x-pack/plugins/security_solution/public/flyout/left/components/related_cases.tsx

```diff
@@ -62,7 +63,19 @@ export const RelatedCases: React.VFC<RelatedCasesProps> = ({ eventId }) => {
   const { loading, error, data, dataCount } = useFetchRelatedCases({ eventId });
 
   if (loading) {
-    return <EuiSkeletonText lines={1} size="m" isLoading={loading} contentAriaLabel="Loading" />;
+    return (
```

Comment:
looking at the other tables under Correlations (the related alerts ones) and looking at the `<EuiInMemoryTable>` below, we're already showing a loading state in the table.
I think we should remove this entire `if` condition. What do you think?

Reply:
will include this in the loading state clean up, good catch!